inc/command/class-command.php

```diff
@@ -41,14 +42,12 @@ protected function configure() {
                                 pass to phpunit. Use the --chassis option
                                 if you are running Local Chassis.
 
-To run Codeception integration tests:
-    codecept [--chassis] -p <path> -b <browser> -o <output-folder> [--] [options]
-                                use `--` to separate arguments you want to
-                                pass to Codeception. Use the --chassis option
-                                if you are running Local Chassis. Use -p path
-                                to specify custom tests folder. Use --browser/-b
-                                to run a headless browser container for acceptance tests,
-                                choose 'chrome', or 'firefox' as needed.
+To run Codeception commands:
+    codecept <subcommand> -p <path> -b <browser> -o <output-folder> <suite-name> [--] [options]
```

Comment:
Optional parameters should be wrapped in square brackets generally, which is pretty much everything after codecept I think:
```suggestion
    codecept [<subcommand>] [-p <path>] [-b <browser>] [-o <output-folder>] [<suite-name>] [--] [options]
```
Is suite name always required? Codeception has a ton of subcommands

Reply:
Nope, should be optional.